catalyst/moodle-mod_facetoface
# Allow users to join sign up for multiple sessions per activity.

### Testing
1. Add a face-to-face activity to a course, make sure "Signup type" is set to multiple
2. As a teacher in the course, create two sessions
3. As a student in the course sign up for one of the sessions
4. Check the course home page, ensure the face-to-face activity shows you are signed up for one session, and the other one is available to join
5. Check the "View all sessions" link and confirm it is consistent
6. Sign up for the other session and verify the "View all sessions" page and the course home page show you signed up for both sessions
7. Cancel one of the sign ups to verify you can cancel and rejoin
